### PR TITLE
Dynamic work section

### PR DESCRIPTION
**Pull request by Ayokunnumi1**
These are the changes implemented in this branch;
* Dynamically created the work section of my html file,
* Added a pop-up-window,
* Made the pop-up-window responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -259,6 +156,74 @@ <h2 class="contact--me">Contact me</h2>
...
+            <a href="">
+              <button>
+                <span>see source</span>
+                <img style="width: 25px;" src="images/github-icon-png.png" alt="">
+              </button>
+            </a>
+          </div>
+        </div>
+      </div>
+  </section>
```
Comment (ridwanediallo) [inline index.html]: - [ ] Great job, team, on making your modal's design look similar to the template and making the open and close buttons work as expected 👏. However, kindly ensure that it is not hard coded in the HTML. The modal should be generated dynamically with Javascript. When the modal opens, it should be prefilled with the information (image, title, description, etc.) that was displayed on the card where we clicked the see project button. 👌
So as stated in the requirements, you need to have a  JavaScript array to store all of the information for all projects and use a loop to go through the array of objects and display the information of a card with the modal whenever a see project button is clicked.  👍️

Please review [the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this task, which I have attached a screenshot below. :point_down: 
#### Task requirement :point_down: 
Screenshot from 2023-06-23 13-24-13

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi team, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Your modal design looks similar to the template. ✔
- Opening and closing the modal is woring correctly. ✔


## Required Changes ♻️

- [ ] Please update the project in the description section from **Hello Microverse** to the real name of the project to make your readme file look more professional. 🛠️🤓
#### Description section from readme file :point_down: 
Screenshot from 2023-06-23 13-47-26



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -253,7 +157,67 @@ <h2 class="contact--me">Contact me</h2>
     </form>
   </section>
   <section class="pop-up" id="modal">
-
```
Comment (abdallahmalima) [inline index.html]: - [ ] 🙏Please update the code to dynamically populate the data for the pop-up display. Currently, it only displays the same data every time you click. This modification will allow users👨‍👩‍👧‍👦 to view the details of each specific project when they click the "See Project" button.

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @Ayokunnumi1 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

A little bit more


 ## Highlights
- Popup appear✔️
- Popup close✔️
- Popup is responsive✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._✔️

**Pull request closed without merging**